### PR TITLE
Split Uptane steps

### PR DESCRIPTION
**Pull request by patriotyk**

### CODE REVIEW

**Comment by patriotyk** [comment]
Just noticed that branch name is wrong. This PR is related to PRO-4478 issue.

File: src/aktualizr.cc [hunk 1]
```diff
@@ -68,8 +68,12 @@ int Aktualizr::run() {
 #endif
   } else {
 #ifdef BUILD_OSTREE
-    // TODO: compile unconditionally
+// TODO: compile unconditionally
+#ifdef BUILD_P11
     boost::shared_ptr<INvStorage> storage = INvStorage::newStorage(config_.storage);
+#else
+    boost::shared_ptr<INvStorage> storage = INvStorage::newStorage(config_.storage);
+#endif
```
Comment (cajun-rat) [inline src/aktualizr.cc]: Aren't both legs of this if the same?

Comment (patriotyk) [reply src/aktualizr.cc]: Oh, yes, I made wrong merge conflict resolve

File: src/invstorage.cc [hunk 2]
```diff
@@ -71,10 +72,10 @@ boost::shared_ptr<INvStorage> INvStorage::newStorage(const StorageConfig& config
         INvStorage::FSSToSQLS(fs_storage, sql_storage);
         return sql_storage;
       }
-      return boost::make_shared<SQLStorage>(config);
+      return boost::shared_ptr<INvStorage>(new SQLStorage(config, p11_config));
     case kFileSystem:
     default:
-      return boost::make_shared<FSStorage>(config);
+      return boost::shared_ptr<INvStorage>(new FSStorage(config, p11_config));
```
Comment (patrickvacek) [inline src/invstorage.cc]: Not a big deal, but probably should keep the `make_shared` in both of these.

**Pull request closed without merging**